go/vt/wrangler/reparent.go

```diff
@@ -1085,3 +1088,7 @@ func (wr *Wrangler) TabletExternallyReparented(ctx context.Context, newMasterAli
 	}
 	return nil
 }
+
+func slaveWasRunning(stopSlaveStatus *replicationdatapb.StopReplicationStatus) bool {
```

Comment:
rename and rewrite to avoid using "slave"

Reply:
Good catch, fixed.